src/components/SystemDetails/GeneralInfo.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 
 import GeneralInformation from '../GeneralInfo/GeneralInformation';
+import useFeatureFlag from '../../Utilities/useFeatureFlag';
 export { default as TextInputModal } from '../GeneralInfo/TextInputModal';
 
 const GeneralInfoTab = (props) => {
-  return <GeneralInformation {...props} />;
+  const enableEdgeImageDetails = useFeatureFlag(
+    'edgeParity.inventory-system-detail'
+  );
+  const enableEdgeInventoryListDetails = useFeatureFlag(
```

Comment:
Yes, it's intentional, we decided to just show this content when the table flag is active too. Either this could work independently but for now as part of parity initiative we want both flags to be active.